README.md

```diff
@@ -8,6 +8,27 @@ This repository contains the OSP C++ library for co-simulations.
 See [`CONTRIBUTING.md`] for contributor guidelines and [`LICENSE`] for
 terms of use.
 
+The libcosim library is demonstrated in [cosim] and [Cosim Demo Application].
+The applications can be downloaded from their release pages: 
+- cosim [releases](https://github.com/open-simulation-platform/cosim-cli/releases)
+- Cosim Demo Application [releases](https://github.com/open-simulation-platform/cosim-demo-app/releases)  
+ 
+How to use
+------------
+To use libcosim in your application you either build the library as described 
+in the section below or you can use conan. As libcosim is made available as a 
+conan package on https://osp.jfrog.io, you can include it in your application
+following these steps:
+
+* Install [Conan] >= 1.7
+* Add the OSP Conan repository as a remote: 
+
+       conan remote add osp https://osp.jfrog.io/artifactory/api/conan/local
+* As our repository has enabled revision support you must set the envorinmental variable `CONAN_REVISIONS_ENABLED=1`
```

Comment:
```suggestion
* As our repository has enabled revision support you must set the environmental variable `CONAN_REVISIONS_ENABLED=1`
```

Reply:
It's also possible to enable revisions via a conan configuration file setting. So instead of being explicit (here and further down) about how, maybe we should just say that revisions must be enabled and link to the relevant part of the Conan docs:
https://docs.conan.io/en/latest/versioning/revisions.html?highlight=revisions#how-to-activate-the-revisions

Reply:
Yes, my first attempt included using conan.conf, but it got too detailed. Including a link tothe Conan doc, was a good idea.